workflow.src

```diff
@@ -0,0 +1,129 @@
+        be part of the commit. This allows you to make  multiple edits at once and split them into different more
+        meaningful/easy-to-review commits.
+      </p>
+      <pre><code>git add -p</code></pre>
+    </li>
+    <li>
+      <p>
+        Commit your staged edits. If you're fixing an issue, reference it in the commit e.g. <code>fix #87</code>
+        (if you write close or fix, it will automatically close the issue once the commit is added in the main branch).
+      </p>
+      <pre><code>git commit -m "&lt;purpose of your edits> (fix #number)"</code></pre>
+      <p><code>git commit --amend</code> allows you to replace the last commit with a new one, if you feel the need to change it.</p>
+    </li>
+    <li>
+      <p>
+        Once all your commits are done, push your branch upstream.
+      </p>
+      <pre><code> git push --set-upstream origin my-wonderful-edits-for-87</code></pre>
+    </li>
+    <li>
+      <p>
+        Using your favorite browser, navigate to the repo on GitHub, e.g.
+      </p>
+      <pre><code>https://github.com/tobie/specs-on-github/</code></pre>
+    </li>
+    <li>
+      <p>
+        Once your authenticated on the repo page, a new button should appear on the page to create a new
+        pull request (<code>Compare &amp; pull request</code>). Go ahead and punch it.
+      </p>
+      <p><img src="img/create-a-pr.png" alt="Compare &amp; pull request" /></p>
+    </li>
+    <li>
+      <p>
+        Make sure you like the title of your pull request, add a proper description (including mentioning the contributors with
+        <a href="https://github.com/w3c/licenses/blob/master/SW-CONTRIBUTING.md">"+@username"</a>), and use
+        <a href="issue-metadata.html">proper GitHub labels</a>, add one or more reviewers so they get pinged.
+        Once you're satisfied, create the pull request. Note that you can update all of that information
+        after that if needed.
+      </p>
+    </li>
+    <li>
+     <p>
+       ALL normative spec changes are generally expected to have a corresponding pull request in
+       <a href="https://github.com/w3c/web-platform-tests/">web-platforms-tests</a>,
+       either in the form of new tests or modifications to existing tests, or must include the rationale for why test
+       updates are not required for the proposed update.
+     </p>
+     <p>
+       Typically, both pull requests (spec updates and tests) will be merged at the same time. If a pull request
+       for the specification is approved but the other needs more work, add the
+       '<a href="https://w3c.github.io/issue-metadata.html">needs tests</a>' label or,
+       in web-platform-tests, the
+       '<a href="https://github.com/w3c/web-platform-tests/issues?utf8=%E2%9C%93&q=label%3Astatus%3Aneeds-spec-decision%20">status:needs-spec-decision</a>'
+       label. Note that a test change that contradicts
```

Comment:
Here, you could document the use of tentative tests, that is tests following the convention `foo.tentative.html`. Such tests can make sense when the spec issue is undecided, but someone is implementing a possible fix and wants to share their tests as they write them.

Reply:
is this documented somewhere that I can link to? Couldn't find it in http://web-platform-tests.org/writing-tests/index.html . My preference would be to mention it with a link to the proper documentation for additional info.

Reply:
I'm adding something in https://github.com/w3c/web-platform-tests/pull/7663

Reply:
That's now merged, should be on http://web-platform-tests.org/writing-tests/file-names.html#test-features once some automatic thing has happened.

Reply:
I added a note in https://github.com/w3c/w3c.github.io/pull/33/commits/ed6d3433c75e342efb99b15b2a1aef2376ca3280

Reply:
As a side, shouldn't file-names.html be mentioned in http://web-platform-tests.org/writing-tests/general-guidelines.html ? Happy to send a PR if you concur.

Reply:
It is linked from the HTTPS section, but could certainly be more prominent. I struggled to find it yesterday myself. PRs welcome :)